javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,10 @@
+/* exported addSuffixToAll */
+
+function addSuffixToAll(words, suffix) {
+  const suffixArray = [];
+  for (let i = 0; i < words.length; i++) {
+    words[i] += suffix;
+    suffixArray.push(words[i]);
```

Comment:
I would combine these lines of code. By doing `words[i] += suffix`, you're adjusting the array value directly, but we don't want to manipulate the array values. We want to reference those values. Here is what I would do instead.

`suffixArray.push(words[i] + suffix)`

With this solution, you're not modifying `words[i]` directly, you're referencing it and creating a new value, and that's what we're pushing.